### PR TITLE
MWPW-134057 | Replace image links in place

### PR DESCRIPTION
**Pull request by aishwaryamathuria**
* If there is an image in Milo with alt text having '|' it is replaced with an a tag and the image element is added inside the a tag. The a tag is then added to the end of the parent element of image. This removes the image from its initial position in the parent element and adds it to the end of the container.
* The change in the PR replaces the image in place. The a tag is added before the image and then the image is shifted inside the 'a' tag keeping the image position same.

Resolves: [MWPW-134057](https://jira.corp.adobe.com/browse/MWPW-134057)

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/drafts/mathuria/mwpw-134057?martech=off
- After: https://miloimglnk--milo--aishwaryamathuria.hlx.page/drafts/mathuria/mwpw-134057?martech=off

### CODE REVIEW

File: libs/utils/utils.js [hunk 1]
```diff
@@ -446,8 +446,9 @@ export function decorateImageLinks(el) {
       if (alt?.trim().length) img.alt = alt.trim();
       const pic = img.closest('picture');
       const picParent = pic.parentElement;
-      const aTag = createTag('a', { href: url, class: 'image-link' }, pic);
-      picParent.append(aTag);
+      const aTag = createTag('a', { href: url, class: 'image-link' });
```
[the remark this answers is not in the document]
Comment (aishwaryamathuria) [reply libs/utils/utils.js:449]: @mokimo Thanks for sharing the feedback. I have pushed a unit test to verify that the the image link is replaced in place.

**Review by ryanmparrish — APPROVED** [review]

**Comment by Ruchika4** [comment]
Hi @aishwaryamathuria , could please address the lint issues in the PR

**Review by ryanmparrish — CHANGES_REQUESTED** [review]
Looks like one too many `../` in the utils path -> test.js file. 

```
import { decorateImageLinks } from '../../libs/utils/utils.js';
```

**Review by chrischrischris — APPROVED** [review]

**Review by ryanmparrish — APPROVED** [review]

**Review by Ruchika4 — APPROVED** [review]

**Review by mokimo — APPROVED** [review]

**Review by sukamat — APPROVED** [review]

**Review by overmyheadandbody — APPROVED** [review]

**Comment by aishwaryamathuria** [comment]
@chrischrischris Could you please confirm if we can merge this PR?

**Pull request merged**